UIExtenderLibModule/UIExtenderLibModule.csproj

```diff
@@ -172,6 +172,8 @@
     <Compile Include="UIExtenderLibModule.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Utils.cs" />
+    <Compile Include="ViewModel\Patches\GauntletPartyScreenCallsites.cs" />
+    <Compile Include="ViewModel\Patches\CharacterDeveloperVMCallsites.cs" />
```

Comment:
For some reason you reference another callsites class that is not part of this commit.

```suggestion

Reply:
See https://github.com/shdwp/UIExtenderLib/pull/1

The file is not added to the project file correctly in this PR. Consequence of including it correctly in my local project version. 
Feel free to PR the change separately.